.github/workflows/ocp-self-runner.yml

```diff
@@ -0,0 +1,111 @@
+name: Test self hosted runner
```

Comment:
This workflow is not the runner. This workflow _uses_ the runner. Wouldn't a better name be something like "Run E2E tests on OCP"?